doc/examples/sprites/lib/main_base64.dart

```diff
@@ -25,7 +24,7 @@ class MyGame extends Game {
   }
 
   @override
-  void update(dt) {}
+  void update(double dt) {}
```

Comment:
I was thinking of this dt vs. t, since we are using both. I vote for that we have them all as dt.

Reply:
agree. but that can be a separate pr :)